.github/workflows/ci.yaml

```diff
@@ -1,7 +1,15 @@
 name: CI
 
-on: [push, pull_request]
-
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v*.**"
+  pull_request:
+    branches: [ master ]
+env:
+  CRATER_VERSION: 1.0.0
```

Comment:
Shouldn't this be the tag name for tags or commit sha etc. for pushes to maser?

Reply:
So this represent current version being worked upon (next release candidate). Every commit to master would derive the tags based on this version PLUS some unique identifier such as commit-sha or run number. Once ready for release, we would push a tag which this version number - that would create a released image artifact. Any further development would then require this version to be bumped (following semantic version major.minor.hotfix)